pennylane/_qubit_device.py

```diff
@@ -359,8 +302,48 @@ def marginal_prob(self, prob, wires=None):
         Returns:
             list[float]: List of the resulting marginal probabilities.
         """
-        wires = wires or range(self.num_wires)
+        wires = list(wires or range(self.num_wires))
```

Comment:
How come this was needed? In the next line ``wires = np.hstack(wires)`` should do a conversion already, correct?

Reply:
I was getting a deprecation warning, the next version of NumPy won't support stacking iterators (like range), only sequences :)

Reply:
Oh cool, very good to know! :)